app/controllers/activities/uploads_controller.rb

```diff
@@ -11,7 +11,7 @@ def new
     authorize report, :show?
 
     @report_presenter = ReportPresenter.new(report)
-    @is_ispf = report.fund.roda_identifier == "ISPF"
+    @is_ispf = report.ispf?
```

Comment:
The pickiest of nits: I think we went with the format `is_thing?` for methods we wrote, to distinguish them somewhat from the `thing?` methods we get for free when using `enum`s. Up to you if you find this a meaningful distinction!

Reply:
I didn't know we had a consistent style, and would favour being consistent with what we have already - will update this